app/decorators/resource_decorator.rb

```diff
@@ -67,6 +67,10 @@ def visibility_badge
     VisibilityBadgeComponent.new(work: self)
   end
 
+  def visibility_badge_mini
+    VisibilityBadgeComponent.new(work: self, mini: true)
+  end
+
```

Comment:
@awead - I don't love declaring a second method for the visibility badge, but wasn't sure if there was a way to pass the `mini` flag into the resource decorator and conditionally set it in the `visibility_badge` method above.

Reply:
Per our conversation with Seth in standup, we are going to show the embargoed badge everywhere and not make a distinction between the mini and non-mini version.